source/apiVolontaria/apiVolontaria/wsgi.py

```diff
@@ -8,9 +8,14 @@
 """
 
 import os
+import sys
 
 from django.core.wsgi import get_wsgi_application
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "apiVolontaria.settings")
+path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Why don't you just replace `apiVolontaria.settings` by `apiVolontaria.local_settings` ?

Reply:
I did, 3 lines under the preview you see here.  
The path was needed for the installation on the server, was not finding apiVolontaria

Reply:
`wsgi` file are operationnal without this new path. You can change the configuration of your server to be operationnal but you can't change this file for your own need.